### ISSUE TITLE
Additional dependencies requiered

### ISSUE THREAD

**Issue by lobis** [mipt-npm/gdml.kt#26]
Most likely this is a mistake on my part due to my little experience with Java/Kotlin/Gradle, but I had trouble adding this repository to my Kotlin project.

After following the readme I add the following to my `build.gradle.kts`:

```
repositories {
    maven("https://repo.kotlin.link")
}

dependencies {
    implementation("space.kscience:gdml:0.3.0")
}
```

If I only add these dependencies I get the following error after running `import space.kscience.gdml.decodeFromString`:

```
Execution failed for task ':compileKotlin'.
> Could not resolve all files for configuration ':compileClasspath'.
   > Could not find net.devrieze:xmlutil-serialization:0.81.0.
     Searched in the following locations:
       - https://jcenter.bintray.com/net/devrieze/xmlutil-serialization/0.81.0/xmlutil-serialization-0.81.0.pom
       - https://repo.maven.apache.org/maven2/net/devrieze/xmlutil-serialization/0.81.0/xmlutil-serialization-0.81.0.pom
       - https://repo.kotlin.link/net/devrieze/xmlutil-serialization/0.81.0/xmlutil-serialization-0.81.0.pom
     Required by:
         project : > space.kscience:gdml:0.3.0 > space.kscience:gdml-jvm:0.3.0

Possible solution:
 - Declare repository providing the artifact, see the documentation at https://docs.gradle.org/current/userguide/declaring_repositories.html
```

Looks like I need additional dependencies. I solved the problem by adding 

```
repositories {
    jcenter()
    maven("https://repo.kotlin.link")
    maven("https://dl.bintray.com/pdvrieze/maven")
}

dependencies {
    implementation("net.devrieze:xmlutil-serialization:0.81.0")
    implementation("space.kscience:gdml:0.3.0")
}
```

From my understanding `jcenter()` is something that would be common enough to be added by default, but the other dependency should be either mentioned on the readme or come bundled with the project somehow.

**Comment by altavir** [comment]
Thanks for the issue. `jcenter` would suffice. It is one of two primary repositories in java ecosystem. Sadly about the month ago they said that they withdraw their services in several months so people are in process of moving to new repositories. I will update the documentation.

**Comment by altavir** [comment]
Fixed in `0.4.0-dev-1`
